lib/gui/models/store.js

```diff
@@ -539,7 +545,7 @@ module.exports = _.merge(redux.createStore(
       // is not very elegant, nor performant, so we decide
       // to intentionally ignore this little flaw since
       // adding extra properties makes no damage at all.
-      return state.set(PERSISTED_PATH, subset);
+      return state.set(PERSISTED_PATH, state.get(PERSISTED_PATH).merge(subset));
```

Comment:
When https://github.com/electron-userland/electron-builder/issues/1674 gets fixed, should both changes to this file be reverted?